manifests/overlays/prod/cost-management/cost-management.enc.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cost-management
```

Comment:
This name needs to match `CronWorkflow`'s `config` parameter. In this case it needs to be `solgate-cost-management`.

Reply:
Fixed.